data_steward/tools/import_rdr_omop.sh

```diff
@@ -70,6 +71,8 @@ source "${TOOLS_DIR}/set_path.sh"
 bq mk -f --description "RDR DUMP loaded from ${RDR_DIRECTORY} dated ${RDR_UPLOAD_DATE}" "${GOOGLE_CLOUD_PROJECT}:${RDR_DATASET}"
 
 python "${DATA_STEWARD_DIR}/cdm.py" "${RDR_DATASET}"
+# accommodate new file "pid_rid_mapping" by RDR
+bq mk --table "${GOOGLE_CLOUD_PROJECT}:${RDR_DATASET}.pid_rid_mapping" "${FIELDS_DIR}/additional_rdr_tables/pid_rid_mapping.json"
```

Comment:
Are we adding labels to tables or just to datasets?

Reply:
As far as I know we haven't added labels to tables, only datasets. But it is something we could look into in the future. This would entail adding capabilities to all our "create table" functions and ddl queries for the addition of labels, which can be a good topic of discussion.

Reply:
Okie dokie! Best we leave it like it is!